source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2174,37 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, PerformBridgeShuntCalEx_Succeeds)
+{
+  EXPECT_THROW_DRIVER_ERROR({
+    const auto AI_MIN = -0.002;
+    const auto AI_MAX = 0.002;
+    create_ai_bridge_chan(AI_MIN, AI_MAX);
+    auto save_response = SaveTaskResponse{};
+    auto status = save_task(save_response);
+  }, MEASUREMENT_TYPE_NOT_SUPPORTED_BY_SELECTED_PHYSICAL_CHANNEL_ERROR);
+
+  EXPECT_THROW_DRIVER_ERROR({
+    auto response = PerformBridgeShuntCalExResponse{};
+    auto status = perform_bridge_shunt_cal_ex(response);
+  }, TASK_SPECIFIED_INVALID_ERROR);
+}
+
+TEST_F(NiDAQmxDriverApiTests, PerformStrainShuntCalEx_Succeeds)
```

Comment:
No, it doesn't succeed. Update the test name and add a brief comment explaining why this is an error test (e.g. // X Series doesn't support shunt cal)